plugin.rb

```diff
@@ -26,10 +25,15 @@ module ::DiscourseCustomTopicLists
         []
       end
     current_user = scope.user
-    custom_lists.select do |list|
-      allowed_groups = list["access"].split(/(?:,|\s)\s*/).map(&:to_i)
-      allowed_groups = [Group::AUTO_GROUPS[:everyone]] if allowed_groups.empty?
-      current_user.in_any_groups?(allowed_groups)
+    raw_custom_lists =
+      custom_lists.select do |list|
+        allowed_groups = list["access"].split(/(?:,|\s)\s*/).map(&:to_i)
+        allowed_groups = [Group::AUTO_GROUPS[:everyone]] if allowed_groups.empty?
+        current_user.in_any_groups?(allowed_groups)
+      end
+    raw_custom_lists.map do |list|
+      list["description"] = PrettyText.cook(list["description"])
+      list
```

Comment:
🏌️ 

```suggestion
    custom_lists.select! do |list|
      allowed_groups = list["access"].split(/(?:,|\s)\s*/).map(&:to_i)
      allowed_groups = [Group::AUTO_GROUPS[:everyone]] if allowed_groups.empty?
      current_user.in_any_groups?(allowed_groups)
    end

    custom_lists.each do |list|
      list["description"] = PrettyText.cook(list["description"])
```

Reply:
I did not know this! thanks for that as well